src/openzaak/components/zaken/models/betrokkenen.py

```diff
@@ -176,12 +176,18 @@ class NietNatuurlijkPersoon(AbstractRolZaakobjectZakelijkRechtRelation):
             "Een uniek nummer toegekend door de Kamer van Koophandel"
         ),
     )
+    vestigings_nummer = models.CharField(
+        max_length=24,
+        blank=True,
+        help_text=mark_experimental("Een korte unieke aanduiding van de Vestiging."),
```

Comment:
The original field on `Vestiging` also has `db_index=True`, might be good to add that here as well since we filter on this field as well